lightning/src/util/events.rs

```diff
@@ -122,6 +123,9 @@ pub enum Event {
 		/// Note that this serves as a payment receipt, if you wish to have such a thing, you must
 		/// store it somehow!
 		payment_preimage: PaymentPreimage,
+		/// The hash which was given to [`ChannelManager::send_payment`].
```

Comment:
`cargo doc` is stupid, if you don't add an extra newline between this and the link ref below, it thinks its all one big doc comment and fails to run.